lib/meson.build

```diff
@@ -84,3 +84,4 @@ libfribidi_dep = declare_dependency(link_with: libfribidi,
   include_directories: incs,
   sources: [fribidi_unicode_version_h, fribidi_config_h],
   compile_args: fribidi_static_cargs)
+meson.override_dependency('fribidi', libfribidi_dep)
```

Comment:
```meson
if meson.version().version_compare('>=0.54.0')
    meson.override_dependency('fribidi', libfribidi_dep)
endif
```

Bumping the version "might" not be wanted. Sure, it's been around for a while. On the other hand, it does drop support for Ubuntu 20.04, an LTS that's likely to still be used for quite a while.  Also, Debian Buster / oldstable (although the backports repository for that has Meson 0.56.0.)

Reply:
You can always install that meson with pip. Takes two seconds.